agent/usershell/usershell_internal_test.go

```diff
@@ -1,13 +1,11 @@
-package usershell_test
+package usershell
```

Comment:
Does this need to be an internal test given the only changes are for exported methods?